WordPress/Classes/ViewRelated/Stats/Shared Views/Stats Detail/SiteStatsInsightsDetailsThirdLevelDelegate.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+/// used when StatsInsightsDetails screen has a 3rd level
+@objc protocol SiteStatsInsightsDetailsThirdLevelDelegate {
+
+    @objc optional func viewMoreSelectedForStatSection(_ statSection: StatSection)
```

Comment:
Is there a reason this can't just go in the existing delegate?

Reply:
Agreed I was trying to get something done for Friday - I looked at this again this weekend and have done it a better + cleaner way.